Example/Tests/Models/Data Sources/AuthorizationsDataSourceSpec.swift

```diff
@@ -237,7 +237,7 @@ class AuthorizationsDataSourceSpec: BaseSpec {
 
                         _ = dataSource.update(with: [finalStatusAuthorization])
 
-                        expect(dataSource.rows).to(equal(1))
+                        expect(dataSource.rows).to(equal(0))
```

Comment:
?

Reply:
Check test. it should be 0 here.

Reply:
Please double check test and read the description of this test. This test failed because of the missing string interpolation Spec Utils